projects/vega-protocol/index.js

```diff
@@ -57,8 +62,19 @@ Object.keys(config).forEach(chain => {
         '0xcb84d72e61e383767c4dfeb2d8ff7f4fb89abc6e': BigNumber(vegaStakedInVesting).plus(BigNumber(vegaStakedInStaking)).toFixed(0)
```

Comment:
okay, I am seeing the same tokens counted in two places (staking & vesting) (?)

The mistake might be on my part for including it in staking, are these tokens in circulation, or they being vested and investors/team get their hands on it for the first time after they get unlocked at certain time?